app/views/layouts/_footer.html.erb

```diff
@@ -3,7 +3,5 @@
     <%= feature('footer-notice') %>
   <% end %>
   <!-- Rails version <%= Rails.version %>-->
-  <% if Rails.env != "production" %>
```

Comment:
Ah, here, we include this because many pages need the API, but we don't provide it in dev mode. In production we set it using a feature. Can we preserve the conditional here? Thanks!